ngscheckmate/1.3/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:18.04
+LABEL maintainer="Miguel Brown (brownm28@chop.edu)"
+
+ENV SAMTOOLS_VERSION 1.9
+ENV BCF_VERSION 1.9
+ENV NCM_HOME /NGSCheckMate
+ENV COMMIT 8ea2c043896adf9bbd1dc5bb2136f3a1dab9b445
+
+RUN apt update && apt install -y wget git python build-essential zlib1g-dev automake libncurses5-dev libbz2-dev liblzma-dev libcurl4-openssl-dev pkg-config \
```

Comment:
Python 2.7 is old.

Reply:
Yeah - this software hasn't been updated since 2019, and I am unsure if they made it python3-compliant...

Reply:
Tried using python3 - it fails